submissions/dashakim/memory-game/script.js

```diff
@@ -93,4 +94,4 @@ function resetBoard() {
   });
 })();
 
-cards.forEach(card => card.addEventListener("click", flipCard));
+section.addEventListener("click", flipCard);
```

Comment:
Well done! Now you have a single event listener with the same game functionality.